src/constant/index.ts

```diff
@@ -1557,6 +1554,8 @@ export const ThemeModes = [
 
 export const imKeyUSBVendorId = 0x096e;
 
+export const SONIC_TESTNET_CHAIN_ID = 64165;
```

Comment:
can you move it to https://github.com/RabbyHub/Rabby/blob/develop/src/ui/views/Ecology/constants.tsx?
Eco should have a separate constant file